.vscode/settings.json

```diff
@@ -23,7 +23,7 @@
   "files.simpleDialog.enable": true,
   "workbench.fontAliasing": "antialiased",
   "workbench.iconTheme": "vs-seti",
-  "workbench.sideBar.location": "right",
+  "workbench.sideBar.location": "left",
```

Comment:
I tried to work with my sidebar on the right but I just don't like it. I agree with this change

Reply:
I agree! Although, I'm confused how this is getting picked up by git 😂